metrics.yaml

```diff
@@ -105,7 +105,7 @@ parameters:
 -
   description: 'Specify version for metrics components; e.g. for "openshift/origin-metrics-deployer:latest", set version "latest"'
   name: IMAGE_VERSION
-  value: "3.4.0"
+  value: "v3.5"
```

Comment:
I had a discussion with tdawson, its v3.5 which is the one which should always work for the 3.5 release. If you set it to 3.5.0 that is only for 3.5.0, not for 3.5.1 nor 3.5.2.

Do we always want this to point to the latest version? or the specific version of a release?